sql/telemetry/client_probe_counts_v1/query.sql

```diff
@@ -177,7 +177,23 @@ RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
   )
 );
 
-WITH bucketed_scalars AS (
+WITH latest_versions AS (
+  SELECT channel, MAX(CAST(app_version AS INT64)) AS latest_version
+  FROM
+    (SELECT
+      normalized_channel AS channel,
+      SPLIT(application.version, '.')[OFFSET(0)] AS app_version,
+      COUNT(*)
+    FROM `moz-fx-data-shared-prod.telemetry.main`
+    WHERE DATE(submission_timestamp) > DATE_SUB(CURRENT_DATE(), INTERVAL 90 DAY)
+    AND normalized_channel IN ("nightly", "beta", "release")
+    GROUP BY 1, 2
+    HAVING COUNT(*) > 1000
```

Comment:
I'm casting a wary eye on this `100` and the `limit 100`. I believe we should increase the value here to say, `1000`, and remove the `limit` altogether.

Reply:
oops! missed removing that limit. nice catch! Thanks! The `having count(*) > 1000` is in the latest commit